wrapping/dockcross-manylinux-build-module-wheels-opencl.sh

```diff
@@ -20,6 +23,12 @@ chmod u+x /tmp/dockcross-manylinux-x64
 
 script_dir=$(cd $(dirname $0) || exit 1; pwd)
 
+mkdir -p $(pwd)/tools
+chmod 777 $(pwd)/tools
+# Build wheels
+mkdir -p dist
```

Comment:
We didn't `cd $(pwd)/tools` first.  Maybe this is right as is, but just checking.

Reply:
Good question, this is correct as-is, ported from ITKPythonPackage scripts: https://github.com/InsightSoftwareConsortium/ITKPythonPackage/blob/master/scripts/dockcross-manylinux-build-module-wheels.sh#L42